src/daemon.py

```diff
@@ -103,11 +104,15 @@ def __execute_yara(self, job: JobId, files: List[str]) -> None:
                     )
```

Comment:
move `num_matches` after the `__update_metadata` call, in case it throws an exception